src/wui/constructionsitewindow.cc

```diff
@@ -537,25 +537,34 @@ bool ConstructionSoldierCapacityBox::handle_key(bool down, SDL_Keysym code) {
 		case ChangeType::kMinus:
 			change_current(-1);
 			return true;
+		case ChangeType::kBigPlus:
+			change_current(3);
+			return true;
+		case ChangeType::kBigMinus:
+			change_current(-3);
+			return true;
 		case ChangeType::kSetMax:
 			set_current(max_);
 			return true;
 		case ChangeType::kSetMin:
 			set_current(min_);
 			return true;
-		default:
-			break;
 		}
 	}
-	return false;
+	return UI::Box::handle_key(down, code);
 }
 bool ConstructionSoldierCapacityBox::handle_mousewheel(int32_t x, int32_t y, uint16_t modstate) {
 	if (!enabled_) {
 		return false;
 	}
 	int32_t change = get_mousewheel_change(MousewheelHandlerConfigID::kChangeValue, x, y, modstate);
 	if (change == 0) {
-		return false;
+		// Try big step
+		change = get_mousewheel_change(MousewheelHandlerConfigID::kChangeValueBig, x, y, modstate);
+		if (change == 0) {
+			return false;
+		}
+		change *= 3;
```

Comment:
You use the constant 3 in multiple places for big step. Please pull out into a `constexpr` variable in some header file (maybe panel.h).

Reply:
Done, but I'm not sure I've found the best possible solution. (I'm never sure about the right scope for `constexpr`s.) I put them in a namespace in `wlapplication_options.h`, next to `get_keyboard_change()`. I first thought I'd try an `enum class` (not sure whether that can act like `constexpr`), but that would need `static_cast`s all the time… yuck. :(

As for all the big steps of 3: Well, I thought of them as mostly independently decided values. :) But that's over then. :)